boards/stm32g031-disco/Makefile.features

```diff
@@ -0,0 +1,10 @@
+CPU = stm32
+CPU_MODEL = stm32g031j6m6
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_rtc
```

Comment:
Looks like `stm32/periph/rtc_all.c` still needs a patch to support STM32G0

You can either try to provide it here, but I'd recommend you to drop the feature for now and provide it as a follow-up PR.